packages/keyring-api/src/eth/v2/eth-keyring-wrapper.ts

```diff
@@ -0,0 +1,213 @@
+import { TransactionFactory, type TypedTxData } from '@ethereumjs/tx';
+import type { MessageTypes, TypedMessage } from '@metamask/eth-sig-util';
+import { SignTypedDataVersion } from '@metamask/eth-sig-util';
+import type { Keyring } from '@metamask/keyring-utils';
```

Comment:
Cosmetic: Would be nice to add `as EthKeyring`, since this `Keyring` interface is truly Ethereum-only?